alacritty/src/renderer/mod.rs

```diff
@@ -441,6 +458,10 @@ impl Batch {
             self.tex = glyph.tex_id;
         }
 
+        let mut cell_flags = RenderingGlyphFlags::empty();
+        cell_flags.set(RenderingGlyphFlags::COLORED, glyph.multicolor != 0);
```

Comment:
Does `multicolor` actually need to be stored in the `Glyph` as a u8? Glyphs itself aren't passed to OpenGL, right?

Reply:
Well, in the current state of things it should be stored there, since that's what we load from rasterizer and the only place where we know that bufffer is RGBA/RGB. They are not passed, but we use them to generate instance data, as you can see here.

Reply:
Yeah, but why not convert them from bool to u8 here? It seems strange that it is stored as u8 on the Glyph?

I'm suggesting this:

```diff
diff --git a/alacritty/src/renderer/mod.rs b/alacritty/src/renderer/mod.rs
index b347556..c66e188 100644
--- a/alacritty/src/renderer/mod.rs
+++ b/alacritty/src/renderer/mod.rs
@@ -123,7 +123,7 @@ pub struct RectShaderProgram {
 #[derive(Copy, Debug, Clone)]
 pub struct Glyph {
     tex_id: GLuint,
-    multicolor: u8,
+    multicolor: bool,
     top: i16,
     left: i16,
     width: i16,
@@ -463,7 +463,7 @@ impl Batch {
             bg_g: cell.bg.g,
             bg_b: cell.bg.b,
             bg_a: (cell.bg_alpha * 255.0) as u8,
-            multicolor: glyph.multicolor,
+            multicolor: glyph.multicolor as u8,
         });
     }
 
@@ -1067,7 +1067,7 @@ fn load_glyph(
         },
         Err(AtlasInsertError::GlyphTooLarge) => Glyph {
             tex_id: atlas[*current_atlas].id,
-            multicolor: 0,
+            multicolor: false,
             top: 0,
             left: 0,
             width: 0,
@@ -1581,7 +1581,7 @@ impl Atlas {
 
         Glyph {
             tex_id: self.id,
-            multicolor: multicolor as u8,
+            multicolor: multicolor,
             top: glyph.top as i16,
             left: glyph.left as i16,
             width: width as i16,
```